### PR TITLE
Added PAPERLESS_CSRF_TRUSTED_ORIGINS with default https url to env

### PR DESCRIPTION
**Pull request by Agent772**
Added the PAPERLESS_CSRF_TRUSTED_ORIGINS to the docke-compose environment variables.
This enables the login via https on the default url the runtipi dashboard provides.

## Summary by CodeRabbit

- **New Features**
  - Introduced a new environment variable for enhanced CSRF security, allowing specification of trusted origins for the application.

- **Security Enhancements**
  - Added configuration to improve protection against cross-site request forgery (CSRF).

- **Version Update**
  - Incremented the version number in the configuration file, indicating potential enhancements or fixes.

### CODE REVIEW

**Comment by meienberger** [comment]
Hello @Agent772 and thank you for the fix! Can you also bump the `tipi_version` in the `config.json` file? Otherwise the update won't be triggered for users

File: apps/paperless-ngx/docker-compose.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ services:
       PAPERLESS_TIKA_ENDPOINT: http://tika:9998
       PAPERLESS_URL: ${APP_PROTOCOL:-http}://${APP_DOMAIN}
       COMPOSE_PROJECT_NAME: paperless-ngx
+      PAPERLESS_CSRF_TRUSTED_ORIGINS: "https:\\paperless-ngx.${LOCAL_DOMAIN}"
```
Comment (meienberger) [inline apps/paperless-ngx/docker-compose.yml:27]: Are the backslashes used on purpose?

Comment (Agent772) [reply apps/paperless-ngx/docker-compose.yml]: nope, that was a typo.
fixed it and bumped tipi version to 42

**Comment by meienberger** [comment]
Thank you @Agent772

**Pull request merged**